### ISSUE TITLE
Uncaught TypeError: this.error_buffer.contains is not a function

### ISSUE THREAD

**Issue by 0x6C38** [baw/scala-worksheet-plus#8]
### Steps to reproduce

1. Open simple scala worksheet file
2. Ctrl + Shift + P -> Run Scala Worksheet Plus: Run

**Atom Version**: 1.13.1
**Electron Version**: 1.4.15
**System**: "Manjaro Linux" (Arch based)
**Thrown From**: [scala-worksheet-plus](https://github.com/imamathwiz/scala-worksheet-plus) package, v0.4.2
**Scala Version**: 2.12.1

### Stack Trace

Uncaught TypeError: this.error_buffer.contains is not a function

```
At /home/myusr/.atom/packages/scala-worksheet-plus/lib/scala-process.coffee:54

TypeError: this.error_buffer.contains is not a function
    at ScalaProcess.module.exports.ScalaProcess.processErr (/home/myusr/.atom/packages/scala-worksheet-plus/lib/scala-process.coffee:54:22)
    at Socket.<anonymous> (/home/myusr/.atom/packages/scala-worksheet-plus/lib/scala-process.coffee:17:41)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at Pipe.onread (net.js:543:20)
```

### Commands

```
     -0:13.4.0 intentions:highlight (input.hidden-input)
     -0:13.3.0 browser-plus:open (input.hidden-input)
     -0:11.1.0 core:close (atom-pane.pane.active)
     -0:04.4.0 command-palette:toggle (input.hidden-input)
     -0:02.6.0 core:backspace (input.hidden-input)
     -0:00.1.0 core:confirm (input.hidden-input)
     -0:00.1.0 scala-worksheet-plus:run (input.hidden-input)
```

### Config

```json
{
  "core": {
    "autoHideMenuBar": true,
    "telemetryConsent": "no",
    "themes": [
      "atom-material-ui",
      "newton-dark-syntax"
    ]
  },
  "scala-worksheet-plus": {}
}
```

### Installed Packages

```coffee
# User
atom-material-syntax, v1.0.2 (inactive)
atom-material-syntax-light, v0.4.6 (inactive)
atom-material-ui, v1.3.9 (active)
atom-neon-ui, v0.37.1 (inactive)
browser-plus, v0.0.87 (active)
Ensime, v0.40.0 (active)
hyperclick, v0.0.40 (active)
intentions, v1.1.2 (active)
language-scala, v1.1.8 (active)
linter, v1.11.21 (active)
neon-syntax, v0.3.7 (inactive)
newton-dark-syntax, v1.1.5 (active)
newton-dark-ui, v1.1.0 (inactive)
scala-worksheet-plus, v0.4.2 (active)
atom-dark-syntax, v0.28.0 (inactive)
atom-dark-ui, v0.53.0 (inactive)
atom-light-syntax, v0.29.0 (inactive)
atom-light-ui, v0.46.0 (inactive)
base16-tomorrow-dark-theme, v1.4.0 (inactive)
base16-tomorrow-light-theme, v1.4.0 (inactive)
one-dark-ui, v1.8.2 (inactive)
one-light-ui, v1.8.2 (inactive)
one-dark-syntax, v1.6.0 (inactive)
one-light-syntax, v1.6.0 (inactive)
solarized-dark-syntax, v1.1.1 (inactive)
solarized-light-syntax, v1.1.1 (inactive)
about, v1.7.2 (active)
archive-view, v0.62.0 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.14.1 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.33.1 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.29.0 (inactive)
autosave, v0.23.2 (active)
background-tips, v0.26.1 (active)
bookmarks, v0.43.2 (active)
bracket-matcher, v0.85.1 (active)
command-palette, v0.39.1 (active)
deprecation-cop, v0.55.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.40.0 (active)
find-and-replace, v0.204.5 (inactive)
fuzzy-finder, v1.4.0 (active)
git-diff, v1.2.0 (active)
go-to-line, v0.31.2 (inactive)
grammar-selector, v0.48.2 (active)
image-view, v0.60.0 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.1 (active)
link, v0.31.2 (inactive)
markdown-preview, v0.159.3 (active)
metrics, v1.1.3 (active)
notifications, v0.65.1 (active)
open-on-github, v1.2.1 (inactive)
package-generator, v1.0.2 (inactive)
settings-view, v0.244.0 (active)
snippets, v1.0.4 (active)
spell-check, v0.68.5 (active)
status-bar, v1.6.0 (active)
styleguide, v0.48.0 (active)
symbols-view, v0.113.1 (inactive)
tabs, v0.103.1 (active)
timecop, v0.33.2 (active)
tree-view, v0.211.1 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.35.1 (active)
whitespace, v0.35.0 (active)
wrap-guide, v0.39.0 (active)
language-c, v0.54.0 (active)
language-clojure, v0.22.1 (active)
language-coffee-script, v0.48.1 (active)
language-csharp, v0.13.0 (active)
language-css, v0.40.1 (active)
language-gfm, v0.88.0 (active)
language-git, v0.15.0 (active)
language-go, v0.43.0 (active)
language-html, v0.47.1 (active)
language-hyperlink, v0.16.1 (active)
language-java, v0.24.0 (active)
language-javascript, v0.122.0 (active)
language-json, v0.18.3 (active)
language-less, v0.29.6 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.37.0 (active)
language-php, v0.37.3 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.1 (active)
language-ruby, v0.70.2 (active)
language-ruby-on-rails, v0.25.1 (active)
language-sass, v0.57.0 (active)
language-shellscript, v0.23.0 (active)
language-source, v0.9.0 (active)
language-sql, v0.25.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.29.1 (active)
language-toml, v0.18.1 (active)
language-xml, v0.34.12 (active)
language-yaml, v0.27.1 (active)

# Dev
No dev packages
```

**Comment by lenalebt** [comment]
Same here, with newer atom version and less packges installed:

### Steps to reproduce

Open simple scala worksheet file
Ctrl + Shift + P -> Run Scala Worksheet Plus: Run

**Atom**: 1.14.1 x64
**Electron**: 1.3.13
**OS**: Ubuntu 16.10
**Thrown From**: [scala-worksheet-plus](https://github.com/imamathwiz/scala-worksheet-plus) package 0.4.2


### Stack Trace

Uncaught TypeError: this.error_buffer.contains is not a function

```
At /home/lena/.atom/packages/scala-worksheet-plus/lib/scala-process.coffee:54

TypeError: this.error_buffer.contains is not a function
    at ScalaProcess.module.exports.ScalaProcess.processErr (/packages/scala-worksheet-plus/lib/scala-process.coffee:54:22)
    at /packages/scala-worksheet-plus/lib/scala-process.coffee:17:41)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at Pipe.onread (net.js:543:20)
```

### Commands

```
     -0:19.9.0 scala-worksheet-plus:run (input.hidden-input)
```

### Non-Core Packages

```
build-sbt 0.6.0 
git-plus 7.2.0 
git-time-machine 1.5.4 
language-scala 1.1.8 
scala-worksheet-plus 0.4.2 
```

**Comment by Aletheia-Lothlorien** [comment]
Same here with latest build.

### Steps to Reproduce

1. Open simple scala worksheet
2. Ctrl + Shift + P -> Run Scala Worksheet Plus: Run

**Atom**: 1.19.0 x64
**Electron**: 1.6.9
**OS**: "Manjaro Linux"
**Thrown From**: [scala-worksheet-plus](https://github.com/imamathwiz/scala-worksheet-plus) package 0.4.2


### Stack Trace

Uncaught TypeError: this.error_buffer.contains is not a function

```
At /home/akhi/.atom/packages/scala-worksheet-plus/lib/scala-process.coffee:54

TypeError: this.error_buffer.contains is not a function
    at ScalaProcess.module.exports.ScalaProcess.processErr (/packages/scala-worksheet-plus/lib/scala-process.coffee:54:22)
    at /packages/scala-worksheet-plus/lib/scala-process.coffee:17:41)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at Pipe.onread (net.js:551:20)
```

### Commands

```
     -0:14.7.0 vim-mode-plus:activate-insert-mode (input.hidden-input)
     -0:10.8.0 command-palette:toggle (input.hidden-input)
     -0:05.1.0 core:confirm (input.hidden-input)
     -0:05.0 scala-worksheet-plus:run (input.hidden-input)
```

### Non-Core Packages

```
atom-alignment 0.13.0 
atom-language-rust 0.12.2 
autocomplete-elixir 1.5.14 
busy-signal 1.4.3 
ex-mode 0.17.0 
fonts 3.0.2 
go-debug 1.5.0 
go-plus 5.5.5 
go-signature-statusbar 1.2.1 
gruvbox-plus-syntax 1.3.1 
hasklig 0.4.0 
hyperclick 0.1.3 
intentions 1.1.5 
language-cpp14 0.6.2 
language-elixir 0.19.0 
language-haskell 1.12.1 
language-mathematica 0.3.2 
language-perl6 1.16.0 
language-rust 0.4.12 
language-scala 1.1.8 
language-scheme 1.4.0 
language-tcltk 0.1.5 
linter 2.2.0 
linter-perl 0.8.1 
linter-rust 0.8.0 
linter-ui-default 1.6.4 
linter-write-good 0.9.0 
nim 0.3.6 
pipe 0.4.0 
platformio-ide-terminal 2.5.5 
racer 0.20.0 
scala-worksheet-plus 0.4.2 
script 3.15.0 
vim-mode-plus 0.96.2 
vim-mode-plus-ex-mode 0.9.1 
vim-mode-plus-keymaps-for-surround 0.2.1 
```

**Comment by baw** [comment]
fixed with 0.5.0 release

Thanks for reporting
